unit-tests/Test/Booster/Pattern/Util.hs

```diff
@@ -47,7 +47,7 @@ app :: Symbol -> [Term] -> Term
 app s = SymbolApplication s []
 
 asConstructor :: SymbolAttributes
-asConstructor = SymbolAttributes Constructor IsNotIdem IsNotAssoc IsNotMacroOrAlias Nothing
+asConstructor = SymbolAttributes Constructor IsNotIdem IsNotAssoc IsNotMacroOrAlias Nothing (Flag False)
```

Comment:
(probably not crucial in tests without LLVM backend but) a constructor won't ever have `hasEvaluators = Flag False` (`True` unless `no-evaluators` is present)